apps/front/src/app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import Image from 'next/image'
+import Logo from '../../../../public/assets/logo.png'
+import SignInPhoto from '../../../../public/assets/signInPhoto.png'
+import Link from 'next/link'
+import LoginForm from '@src/app/(auth)/login/loginForm/loginForm'
+import { InputAdornment, TextField } from '@mui/material'
+import ClipBoardCopy from '@src/components/ui/copyToClipboard'
+
+const Login = () => {
+    return (
+        <div className='min-h-screen text-white font-raleway flex justify-center items-center bg-black'>
```

Comment:
aby korzystać sensownie z tailwinda to ostylowane komponenty powinny być wydzielane osobnych plików, tak to takie klasy będą się powtarzać wielokrotnie w wielu miejsach w codebase